src/dot-merlin/dot_merlin_reader.ml

```diff
@@ -427,6 +427,7 @@ let postprocess cfg =
         (dirs :> Merlin_dot_protocol.directive list)
       )
     ; (cfg.pass_forward :> Merlin_dot_protocol.directive list)
+    ; cfg.stdlib |> Option.map ~f:(fun stdlib -> `STDLIB stdlib) |> Option.to_list
```

Comment:
I think this results in incorrect behavior, as the `-ocamllib-path` flag will be ignored. With this change, it will appear to `mconfig_dot.ml` as if the `.merlin` file contained the `STDLIB` directive, which it will then prioritize over the `-ocmallib-path` flag. Indeed, I added a test for this case and confirmed that accepting this change would break the test.